### ISSUE TITLE
'aria-toggle-field-label' - Rule issue

### ISSUE THREAD

**Issue by kushalmatalia1512** [dequelabs/axe-core#2032]
Expectation: << Rule ID should be - 'aria-toggle-field-label'
                                                  OR - 'aria-toggle-field-name' >>

Actual: << Current Rule ID is - 'aria-toggle-field-name' >>

Motivation: << When we use the **https://cdnjs.cloudflare.com/ajax/libs/axe-core/3.4.2/axe.js**,
                                    the Rule Id is 'aria-toggle-field-name', but
                                    in **https://dequeuniversity.com/rules/axe/3.4**, there is no such Rule as 
                                    per 'aria-toggle-field-name' Id, instead it is mentioned there as 
                                    'aria-toggle-field-label' and also in the description of this Rule Id
                                         **https://dequeuniversity.com/rules/axe/3.4/aria-toggle-field-label**,
                                    NO details, descriptions, can be found as of today.

                                   Can i get any confirmation on this, and whether if my observation is correct or 
                                   do i need to follow some documentations?
                                   
<img width="646" alt="Axe1" src="https://user-images.githubusercontent.com/26283886/74142230-73938900-4c1e-11ea-99a0-13667c49431c.PNG">
wrong>>

<pre><code>
axe-core version: 3.4.2
DequeUniversity List of Axe Rules - 3.4

Browser and Assistive Technology versions

For Tooling issues:
- Node version: XX  
- Platform:  
</code></pre>

**Comment by straker** [comment]
Thanks for letting us know.

**Comment by jeankaplansky** [comment]
https://dequeuniversity.com/rules/axe/3.4/aria-toggle-field-name fixed and verified as of 4:49 pm EST 2/12/2020

**Comment by kushalmatalia1512** [comment]
Thanks.
Appreciated.
